Sources/C7.swift

```diff
@@ -1,7 +1,83 @@
+#if os(Linux)
+import Glibc
+#else
+import Darwin.C
+var mtid = mach_timebase_info_data_t()
+#endif
+
 #if swift(>=3.0)
 #else
 public typealias ErrorProtocol = ErrorType
 public typealias RangeReplaceableCollection = RangeReplaceableCollectionType
 public typealias MutableCollection = MutableCollectionType
 public typealias Sequence = SequenceType
 #endif
+
+public var now: Int {
+    #if os(Linux)
+    var ts = timespec()
+    clock_gettime(CLOCK_MONOTONIC, &ts)
+    return Int(ts.tv_sec) * 1000 + Int(ts.tv_nsec) / 1000000
+    #else
+    if mtid.denom == 0 {
+        mach_timebase_info(&mtid)
+    }
+    let ticks = mach_absolute_time()
+    return Int(ticks * UInt64(mtid.numer) / UInt64(mtid.denom) / 1000000)
+    #endif
+}
+
+public extension Int {
+    /// Interval of `self` from now.
+    var fromNow: Int {
```

Comment:
Tanner had some good logic on a different PR about functions being a bit more clear about computation. I'd personally prefer `func secondsFromNow() -> Int`. It also reads nicely as `let t = 300.secondsFromNow()`

I'm fine w/ what's here if others disagree.